### DISCUSSION TITLE
PnP Search with Micorsoft Search as Data Source shows Disabled AAD user profile

### DISCUSSION THREAD

**Discussion by Ami99049** [microsoft-search/pnp-modern-search#3221] [Q&A]
Dear Team,

I am using Microsoft Search as a Data Source in PnP. When I search for a user which is disabled in AAD that user is not coming in PnP search webpart which is absolutely perfect but when I search that user's manager then in the hover card that disabled user is still present in org chart. Is there any way to exclude the Disabled User from Hover Card too.

**Comment by wobba** [comment]
We don't control how the graph toolkit works and you can ask over at https://github.com/microsoftgraph/microsoft-graph-toolkit
